hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/transaction/TransactionRollbackMessageTask.java

```diff
@@ -74,7 +74,7 @@ public String getDistributedObjectName() {
 
     @Override
     public String getMethodName() {
-        return null;
+        return "transactionRollback";
```

Comment:
Similarly this should be rollbackTransaction

Reply:
changed.